src/test/elements/netsuitehcv2/subsidiaries.js

```
@@ -0,0 +1,19 @@
+'use strict';
+
+const suite = require('core/suite');
+const chakram = require('chakram');
+const expect = chakram.expect;
+
+suite.forElement('humancapital', 'subsidiaries', (test) => {
+  	test.should.supportS();
+	  test.withOptions({ qs: { page: 1, pageSize: 5}}).should.supportPagination();
+    test
+      .withOptions({ qs: { where: `email = 'test@gamil.com' ` } })
```

can you remove the hardcoded values, since, this can fail in other sandboxes

---

updated the script